.drone.yml

```diff
@@ -17,6 +17,24 @@ steps:
   - cargo test
   - echo "HOME FOLDER = " $HOME
 
+---
```

Comment:
why one more pipeline and not use the existing integration test one?

Reply:
it's one of steps for multiple VM test.

also there will be more complex test using multiple VMs.
to test many VMs, i need to more control in language level.

Reply:
is multi-VM setup necessary for testing HD-wallet functionality?

Reply:
not necessary,
it's one of steps to test jailing, unjailing